openpype/hosts/maya/plugins/create/create_look.py

```diff
@@ -22,4 +22,6 @@ def __init__(self, *args, **kwargs):
         self.data["maketx"] = self.make_tx
 
         # Enable users to force a copy.
-        self.data["forceCopy"] = True
+        # - on Windows is "forceCopy" always changed to `True` because of
+        #   windows implementation of hardlinks
```

Comment:
If the fix is temporary and we log it to the end user (see below in the code) we might be fine with not having this comment in the Creator code. I'm fine with either.